langfuse/src/langfuse.ts

```diff
@@ -22,12 +23,22 @@ export class Langfuse extends LangfuseCore {
   private _storageCache: any;
   private _storageKey: string;
 
-  constructor(params: { publicKey: string; secretKey: string } & LangfuseOptions) {
-    super(params);
-    const { publicKey, secretKey, ...options } = params;
+  constructor(params?: { publicKey?: string; secretKey?: string } & LangfuseOptions) {
+    const { publicKey, secretKey, ...options } = utils.configLangfuseSDK(params);
+    if (!secretKey) {
+      throw new Error("secretKey is required");
+    }
+    utils.assert(secretKey, "[Langfuse] secretKey is required for instantiation");
```

Comment:
utils.assert does exactly the same thing as if(!secretKey) before, remove both (see above) or remove one of them if we want to keep this behavior (throw not log if keys are missing)